services/routes/civic.js

```diff
@@ -1,12 +1,11 @@
 const https = require('https');
 const axios = require('axios');
 const apiKey = process.env.NODE_GOOGLE_CIVIC_API_KEY;
-const currentElectionId = 7000;
-
-//axios.interceptors.request.use((request) => {
-//  console.log('Starting Request', JSON.stringify(request, null, 2));
-//  return request;
-//});
+//const currentElectionId = 7000; // 2020 US Presidential Election
+//const currentElectionId = 2000; // VIP Test Election
+//const currentElectionId = 5029; // Louisiana General Congressional Election
+const currentElectionId = process.env.NODE_CIVIC_ELECTION_ID || 2000;
```

Comment:
Environment variable that can be updated for deployment. For 2020 GA senate runoffs it will be set to 7001 which is the Civic APIs id for this election. The local dev id is 2000 which return test data from the Civic API.